database/tables/submission_grading_context_embeddings.pg

```diff
@@ -0,0 +1,12 @@
+columns
+    embedding: vector(1536)
+    id: bigint not null default nextval('submission_grading_context_embeddings_id_seq'::regclass)
+    submission_id: bigint not null
+    submission_text: text
```

Comment:
Is this nullable? If not, please add an appropriate `NOT NULL` constraint.

Reply:
Nothing should be nullable here. We just made the updates.